ext/yajl/yajl_encode.c

```diff
@@ -76,10 +82,39 @@ yajl_string_encode2(const yajl_print_t print,
             case '\b': escaped = "\\b"; break;
             case '\t': escaped = "\\t"; break;
             case '/':
-              if (htmlSafe) {
+              if (htmlSafe == 1) {
```

Comment:
Lets change this to `htmlSafe == 1 || htmlSafe == 2` to address @rafaelfranca's comment